angular/src/components/set-password.component.ts

```diff
@@ -69,30 +82,51 @@ export class SetPasswordComponent extends BaseChangePasswordComponent {
 
     async performSubmitActions(masterPasswordHash: string, key: SymmetricCryptoKey,
         encKey: [SymmetricCryptoKey, EncString]) {
+        // Create Set Password Request
```

Comment:
I would say this makes for a better function than comment. We should create a constructor for SetPasswordRequest.

Reply:
Created constructor.